server/lib/live-blogs/render.js

```diff
@@ -17,7 +20,7 @@ ${dateTransform(data.emb * 1000, {classname: 'live-blog--time', format: 'datetim
 	${data.authornamestyle === 'initials' ? data.author : data.authordisplayname}
 </span>
 
-<p>${data.textrendered}</p>
+<p>${rewriteAlphavilleURL(data.textrendered)}</p>
```

Comment:
is this just for emoticons? we already have https://github.com/Financial-Times/google-amp/blob/master/server/lib/transforms/fix-emoticons.js